web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -334,16 +335,16 @@ was not inplace, since it is possible to go out of memory because of this.
 
 #### Return the calling object (`self`) also when using `inplace=True`?
 
-The downsides of keeping the `inplace=True` option for certain methods, are that the return type of those methods will
-now depend on the value of `inplace`, and that method chaining will no longer work.
-
-One way around this is to have the method return the original object that was operated on inplace when `inplace=True`.
+One of the downsides of the `inplace=True` option is that the return type of those methods
+depends on the value of `inplace`, and that method chaining does not work.
+Those downsides are still relevant for the cases where we keep `inplace=True`. Therefore,
```

Comment:
The wording in the last sentence feels a little verbose to me.

I'd reword it to "To address this, we can have those methods return the object that was operated on inplace when `inplace=True`".

Reply:
Thanks, used your suggestion.